### ISSUE TITLE
Throws KeyNotFoundException when getting the count of drawings.

### ISSUE THREAD

**Issue by robincodex** [EPPlusSoftware/EPPlus#1673]
### EPPlus usage

Personal use

### Environment

Windows

### Epplus version

7.4.0

### Spreadsheet application

Microsoft365 Excel

### Description

This error appears after version 7.4.0, version 4.3.2 doesn't have this problem.

[test.xlsx](https://github.com/user-attachments/files/17676730/test.xlsx)

This error occurs when `sheet.Drawings.Count`
```
System.Collections.Generic.KeyNotFoundException: The given key '' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at OfficeOpenXml.Packaging.ZipPackageRelationshipCollection.get_Item(String id)
   at OfficeOpenXml.Packaging.ZipPackagePartBase.GetRelationship(String id)
   at OfficeOpenXml.Drawing.ExcelPicture..ctor(ExcelDrawings drawings, XmlNode node, ExcelGroupShape shape)
   at OfficeOpenXml.Drawing.ExcelDrawing.GetDrawingFromNode(ExcelDrawings drawings, XmlNode node, XmlElement drawNode, ExcelGroupShape parent)
   at OfficeOpenXml.Drawing.ExcelDrawing.GetDrawing(ExcelDrawings drawings, XmlNode node)
   at OfficeOpenXml.Drawing.ExcelDrawings.AddDrawings()
   at OfficeOpenXml.Drawing.ExcelDrawings..ctor(ExcelPackage xlPackage, ExcelWorksheet sheet)
   at OfficeOpenXml.ExcelWorksheet.LoadDrawings()
   at OfficeOpenXml.ExcelWorksheet.get_Drawings()
```

**Comment by swmal** [comment]
Thanks for reporting, we will have a look at this.

**Comment by OssianEPPlus** [comment]
That's because we only recently added support for Linked Images, before we did not check the "linked" property.
Your file has a Link attribute but it contained no value.
Drawings1.xml:
![image](https://github.com/user-attachments/assets/ebbf897c-de44-4f54-b7be-f9d84cbab591)

We have added a fix for this. It should be in [Our NuGet Develop Feed](https://github.com/EPPlusSoftware/EPPlus/wiki/Using-our-develop-Nuget-feed) within a few days.

Out of curiosity, how did you add your image/create your workbook?
Microsoft365 Excel does not generate xml that way when I attempt to.

**Comment by robincodex** [comment]
> That's because we only recently added support for Linked Images, before we did not check the "linked" property. Your file has a Link attribute but it contained no value. Drawings1.xml: ![image](https://private-user-images.githubusercontent.com/122265629/388532122-ebbf897c-de44-4f54-b7be-f9d84cbab591.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIyMTA2NDEsIm5iZiI6MTczMjIxMDM0MSwicGF0aCI6Ii8xMjIyNjU2MjkvMzg4NTMyMTIyLWViYmY4OTdjLWRlNDQtNGY1NC1iN2JlLWY5ZDg0Y2JhYjU5MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyMVQxNzMyMjFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03MGZjNjdhODk1OTJmNmQ3ODE5ZjI1OWZlZDcxYjliODFjNjIzYjIzYzM3YWMwNzdjZTFmMzA4NDM3MGJiMmM0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.KwWYGeof8XSGzflHAeiNeXY927lmUXdyFKUCqgbYz94)
> 
> We have added a fix for this. It should be in [Our NuGet Develop Feed](https://github.com/EPPlusSoftware/EPPlus/wiki/Using-our-develop-Nuget-feed) within a few days.
> 
> Out of curiosity, how did you add your image/create your workbook? Microsoft365 Excel does not generate xml that way when I attempt to.

This file was exported from Tencent Cloud Documents, similar to Google Docs, so it seems I shouldn't have given feedback on this issue, it's an error caused by Cloud Documents not adhering to standards.

Thank you for resolving this; I apologize for any inconvenience caused.

**Comment by JanKallman** [comment]
Fixed in EPPlus 7.5.2
